### PR TITLE
FAQ Section, Docs contributing link, and Hugo Windows Info

### PR DESCRIPTION
**Pull request by drewf7**
Hi! Loving this project.

Changes:
- Add FAQ Section
- Add FAQ For How To Force Rescans
- Add Link Under Development Setup To This Repo (Because I initially thought this would be in delun/navidrome)
- Add A Warning in the README to help anyone else looking to run this repo on a windows machine

I started out tonight looking for a way to force a rescan of existing albums. I couldn't find information on the docks site, but I did find an answer on reddit that linked to issue 130.

I thought it might be helpful to add that information to the docs site. I hit a bit of a snag building on windows. So I also added some help text for that.

Happy to make changes if you want.

Thanks.

### CODE REVIEW

**Comment by deluan** [comment]
Hey, thanks for the contribution! I'll review the changes and let you know

File: content/en/docs/Developers/dev-environment.md [hunk 1]
```diff
@@ -13,6 +13,9 @@ If you are stuck or have any questions, please join our [Discord server](https:/
 give us a shout on the `#dev` channel
 {{% /pageinfo %}}
 
+### Docs Contributions
```
Comment (deluan) [inline content/en/docs/Developers/dev-environment.md:16]: Not sure if this is the best place to add this information. Do you really think it is necessary, given that all pages have links to the documentation project:
Screen Shot 2020-10-05 at 7 38 31 PM

**Review by deluan — CHANGES_REQUESTED** [review]
Sorry for the delay in reviewing this. Great contributions, just not sure about the dev environment. See below

File: content/en/docs/Developers/dev-environment.md (hunk 1 above)
Comment (drewf7) [reply content/en/docs/Developers/dev-environment.md]: Fair point! I went ahead and pulled that change.

Let me know if there's anything else.

Thanks

Comment (deluan) [reply content/en/docs/Developers/dev-environment.md]: LGTM

**Pull request merged**